setup.py

```diff
@@ -24,7 +24,7 @@
     project_urls={
         'Documentation': 'https://docs.asf.alaska.edu/asf_search/basics/'
     },
-    packages=find_packages(),
+    packages=find_packages(exclude=["tests.*", "tests"]),
```

Comment:
This is a good addition, and for consideration, the same comment as above: maybe consider adding /examples